crates/stress/src/bin/sim2h_stress.rs

```diff
@@ -10,18 +10,17 @@ extern crate lib3h_sodium;
 #[macro_use]
 extern crate log;
 extern crate serde;
+#[macro_use]
+extern crate serde_derive;
 extern crate serde_json;
 extern crate sim2h;
 extern crate structopt;
+extern crate toml;
```

Comment:
with `edition = "2018"` extern crate is no longer needed

```suggestion
```

Reply:
oh, snap. I'm gonna stop writing those then!